app-office/texstudio/additional-files/texliveinstall.sh

```diff
@@ -0,0 +1,36 @@
+#/bin/sh
+
+pkgman install texlive_core -y
```

Comment:
This can be removed, it is in REQUIRES for the other texlive packages. It's name would be `texlive_core_x86` on x86.

Reply:
hmm you are right here, need to find a way to make this do it's job on 32bit (need to boot that more often lately) :)

Reply:
Ohwell, swithing with "setarch" fixes installing on 32bit.